python/cuml/metrics/__init__.py

```diff
@@ -15,6 +15,6 @@
 #
 
 from cuml.metrics.trustworthiness import trustworthiness
-from cuml.metrics.regression import r2_score
+from cuml.metrics.regression import r2_score, mean_squared_error
```

Comment:
After chatting with a few folks lately, I think doing one import per line in the init (and in general) is much better

Reply:
Done.